lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -201,10 +199,70 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
         calculateConvPoolOutputDims(idim.h, idim.w, kernels, strides, pads);
     std::array<size_t, 4> outDims = {
         {idim.n, outSz.first, outSz.second, depth}};
-    auto outTy = G_.getParent()->uniqueType(ElemKind::FloatTy, outDims);
+
+    TypeRef outTy;
+    Constant *filter;
+    Constant *bias;
+    if (typeName == "Conv") {
+      // Construct the Bias field.
+      Tensor biasTensor(ElemKind::FloatTy, {depth});
+      biasTensor.zero();
+
+      // Check if we have a serialized bias vector.
+      if (op.input_size() > 2) {
+        auto &biasTensorName = op.input(2);
+        if (tensors_.count(biasTensorName)) {
+          // Load the serialized bias vector.
+          Tensor *b = getTensorByName(biasTensorName);
+          biasTensor.assign(b);
+        }
+      }
+      outTy = G_.getParent()->uniqueType(ElemKind::FloatTy, outDims);
+      filter = G_.getParent()->createConstant("conv.filter", wtag);
+      bias = G_.getParent()->createConstant("conv.bias", biasTensor);
+    } else {
+      assert(dict.count("Y_zero_point") &&
+             "missing zero point for quantzied type");
```

Comment:
can you be more explicit that the problem is with zero point for the `bias` param

Reply:
It has been explained before defining OFFSETSHIFT if it is what you mean:).
```
/// For the quantized Caffe2 ops, the activations are quantized to uint_8.                                                                                                    
/// In Glow, the activations are quantized to int_8. Therefore, for the offset                                                                                                
/// read from quantized caffe2 model, we need to subtract 128(i.e. INT8_MIN) to                                                                                               
/// make the activations becomes int8_t.                                                                                                                                      
/// For Glow: -127 <= orig_fp32/scale_1 + offset_1 < 128                                                                                                                      
/// For Caffe2: 0 <= orig_fp32/scale_2 + offset_2 < 255                                                                                                                       
/// Therefore, we can make scale_1 == scale_2, and offset_1 = offset2 - 128                                                                                                   
const int32_t OFFSETSHIFT = 128
```;

Reply:
yeah, i was referring to `"missing zero point for quantzied type"`
just to clarify that it's for the output type etc.

Reply:
Thanks! :)